.github/workflows/release.yml

```diff
@@ -10,94 +10,6 @@ env:
   THIRD_PARTY_GIT_AUTHOR_NAME: nr-opensource-bot
 
 jobs:
-  generate-uuid:
```

Comment:
We will need to keep this workflow for now, the service does not generate the UUID on the server side. This is mainly due to it running on multiple environments and the UUID needing to be consistent across them

Reply:
this just moves the job to the new pr-merged workflow, it will still generate and commit UUIDs